app/javascript/layouts/buttons.css

```diff
@@ -1,13 +1,24 @@
 @layer components {
   /* Prefixed with "top" due to Bootstrap conflicts */
   .top-btn {
-    @apply inline-block rounded shadow py-4 px-10 md:px-14 border-0
+    @apply inline-block rounded shadow py-3.5 px-10 md:px-14 border-0
     text-base font-bold text-center no-underline text-white
     cursor-pointer
     hover:bg-opacity-80
-
   }
   .top-btn-primary {
     @apply bg-odin-green
   }
+  .top-btn-danger {
+    @apply bg-red-500
+  }
+  .top-btn-disabled {
+    @apply bg-gray-300 text-gray-500 hover:text-black
+  }
+  .top-btn-secondary {
+    @apply bg-gold
+  }
+  .submissions-button {
```

Comment:
These buttons are looking a little big, Can we give them `py-2` for padding-top and bottom?